source/dub/dependency.d

```diff
@@ -412,6 +455,7 @@ struct Dependency {
 		A specification is valid if it can match at least one version.
 	*/
 	bool valid() const {
+		if (this.isGit) return true;
```

Comment:
do these `isGit` calls make sense for only git? I think having a generic `isSCM` method (which for now would do the same thing as isGit) would make this code clearer